### ISSUE TITLE
BannedApiAnalyzers not banning System.Web namespace

### ISSUE THREAD

**Issue by bwilliams1** [dotnet/roslyn-analyzers#7011]
### Analyzer

**Diagnostic ID**: RS0030

### Analyzer source

**NuGet Package**: [[Microsoft.CodeAnalysis.NetAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers)](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BannedApiAnalyzers/)

**Version**: 3.3.4 (Latest)

### Describe the bug

adding N:System.Web isn't causing green squiggles or compiler warnings.

### Steps To Reproduce

1, create a .net4.8 project with BannedApiAnalyzers 
1. add N:System.Web to a BannedSymbols.txt file
2. add System.Web namespace
3. run a build
4. System.Web namespace isn't flagged

### Expected behavior
System.Web namespace is flagged
### Actual behavior
System.Web namespace isn't flagged
## Additional context
added a banned type to prove everything is wired correctly. 
![image](https://github.com/dotnet/roslyn-analyzers/assets/9018609/aa7f33ca-0fc0-4c05-9719-4d0f7d481dc5)
![image](https://github.com/dotnet/roslyn-analyzers/assets/9018609/23e9bfea-8285-4f2f-a2f1-18e2c332c27e)

**Comment by twsouthwick** [comment]
@bwilliams1 looks like this was added in https://github.com/dotnet/roslyn-analyzers/pull/6521. 3.3.4 looks to have been released 9 months ago, which would be before that PR merged. There are a number of prerelease versions out there since the PR landed - can you try the latest out?

**Comment by bwilliams1** [comment]
> @bwilliams1 looks like this was added in #6521. 3.3.4 looks to have been released 9 months ago, which would be before that PR merged. There are a number of prerelease versions out there since the PR landed - can you try the latest out?

I upgraded to <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" Version="3.11.0-beta1.23472.1">

I still don’t see the warning

**Comment by twsouthwick** [comment]
did you restart VS? I often need to do that when updating analyzers to make sure they show up

**Comment by bwilliams1** [comment]
> did you restart VS? I often need to do that when updating analyzers to make sure they show up

yes. I tried it fresh this morning and didn't see it trigger the warning.

**Comment by bwilliams1** [comment]
looks like the warning will appear on "Invocations" on types within the namespace. I was expecting a warning on the using statement itself.

To see it I had to add another invocation. `request.AcceptTypes`. My other "type" rule was taking precedence over what the "namespace" rule would have also flagged.

```
N:System.Web;this is from System.Web rule
T:System.Web.HttpContext;this is from HttpContext rule
```

![image](https://github.com/dotnet/roslyn-analyzers/assets/9018609/2c31909d-8a8f-4eeb-b32a-b549862fec08)
